go/client/cmd_tlf.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"encoding/json"
+	"os"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
+)
+
+func newCmdTlf(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "tlf",
+		Usage: "Debug TLF related information",
+		Subcommands: []cli.Command{
+			newCmdTlfCryptKeys(cl, g),
+		},
+	}
+}
+
+type cmdTlfCryptKeys struct {
+	libkb.Contextified
+	tlfName string
+}
+
+func newCmdTlfCryptKeys(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "crypt-keys",
+		Usage:        "List crypt keys of all generations for a given TLF",
+		ArgumentHelp: "<tlf name>",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&cmdTlfCryptKeys{Contextified: libkb.NewContextified(g)}, "crypt-keys", c)
+		},
+	}
+}
+
+func (c *cmdTlfCryptKeys) Run() error {
+	tlfClient, err := GetTlfClient(c.G())
+	if err != nil {
+		return err
+	}
+
+	if err = RegisterProtocols(nil); err != nil {
```

Comment:
It seems it's actually what `RegisterProtocols` is doing: https://github.com/keybase/client/blob/e84f240a9e89fce5b264399b37f6f85c78c09e30/go/client/rpc.go#L108

Or did you mean using `RegisterProtocolsWithContext`?